README.md

```diff
@@ -23,16 +23,56 @@ Follow the inital project setup for non-engineers described in [Notion](https://
+
+### Local Setup
+
+If, for some reason, you are unable to use Docker, you can still set up the app locally. Note that you can set up each of these three services separately, but some functionality make not work. For example, if only the client is running, none of the calls to the backend or geo-service service will work. If only the backend is running, none of the calls to the geo-service will work.
+
+#### Client
+
+1. Install [nvm](https://github.com/nvm-sh/nvm) (for Mac/Linux), or [nvm-windows](https://github.com/coreybutler/nvm-windows) (for Windows).
+1. Using nvm, install Node 12.16.2: `nvm install 12.6.2`.
+1. Enter the `client` directory and run `npm install`.
+1. Copy the `.env.example` in the `client` directory to `.env`.
+1. Replace `TODO` entries in the `.env` with correct values (this is not needed to run the project in a responding but non-functional state). Consult the [non-engineer guide](https://www.notion.so/fightpandemics/Instructions-for-UI-testing-for-non-engineers-26d1237683d649f1a45f01e1b5a6c24b) for AIRTABLE variable values.
+1. Start the client app by running `npm start`, and wait for the app to start up. (`Starting the development server...` is not the final line).
+1. Finally, navigate to [localhost:3000](http://localhost:3000) in your browser - the page title should be "Fight Pandemics" and you should see a styled page.
+
+#### Backend
+
+1. If not already installed, install nvm and Node 12.6.2 (see steps 1 and 2 in the Client section above).
+1. Enter the `backend` directory and run `npm install`.
+1. Copy the `.env.example_local` in the `backend` directory to `.env`.
+1. Replace `TODO` entries in the `.env` with correct values (this is not needed to run the project in a responding but non-functional state). Ask in the `#engineering` Slack channel for AUTH variable values.
+1. Start the backend server by running `npm start`. NOTE: If you want the server to automatically restart on code changes, run `npm run dev` instead.
+1. The backend can be accessed at `localhost:8000` using cURL, Postman, or a similar API testing tool.
+
+#### Geo Service
+
+1. Install [pyenv](https://github.com/pyenv/pyenv) (for Max/Linux) or [pyenv-win](https://github.com/pyenv-win/pyenv-win) (for Windows).
+1. Using pyenv, install Python 3.7.6: `pyenv install 3.7.6`
+1. Enter the `geo-service` folder, and create a virtual environment: `python -m venv venv`.
+1. Activate the [virtual environment](https://docs.python.org/3/library/venv.html): `. venv/bin/activate` (Mac/Linux); `venv\Scripts\activate.bat` (Windows CMD); `venv\Scripts\Activate.ps1` (Windows PowerShell).
+1. Run `pip install -r requirements.txt` to install dependencies.
+1. Start the geo-service by running `python app.py`
+1. The geo-service can be accessed at `localhost:5000` using cURL, Postman, or a similar API testing tool.
+
```

Comment:
I just realized that setup instructions for MongoDB is missing. I will add this shortly.